mautic/mautic
# [UI] Replace instance icon with Campaign icon

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses the need for clearer visual representation in the Campaign Forms section. Currently, the "instance" icon may confuse users, as it doesn't accurately reflect the purpose of the forms. By replacing it with a Campaign (megaphone) icon, we enhance user understanding and navigation.

Also use lined icons for the settings menu to standardize icons on dropdowns.




| Before                                 | After
| -------------------------------------- | ---
|   ![image](https://github.com/user-attachments/assets/cd902662-db36-41b8-8c32-af0cd9e48313) ![image](https://github.com/user-attachments/assets/ea566dac-cb56-440b-871f-a198abd600ae)                                      | ![image](https://github.com/user-attachments/assets/5600e1a3-c1d3-4bf2-8ae0-c2406fa320b9)  ![image](https://github.com/user-attachments/assets/bc8cc3ca-ab78-4295-9212-707502f647c2) ![image](https://github.com/user-attachments/assets/5600e1a3-c1d3-4bf2-8ae0-c2406fa320b9)

---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Forms page again, create a Campaign Form, save it, and then close it. Go back to the list and verify that the Campaign (megaphone) icon is displayed.
